libdevcore/CommonJS.h

```diff
@@ -48,12 +48,14 @@ inline std::string toJS(dev::bytes const& _n)
 
 /// Convert string to byte array. Input parameters can be hex or dec. Returns empty array if invalid input e.g neither dec or hex.
 bytes jsToBytes(std::string const& _s);
-/// Add '0' on the head of _b until _l.
+/// Add '0' on the head of @a _b until @a _l.
 bytes padded(bytes _b, unsigned _l);
+/// Add '0' on the queue of @a _b until @a _l.
+bytes paddedRight(bytes _b, unsigned _l);
 /// Removing all trailing '0'. Returns empty array if input contains only '0' char.
 bytes unpadded(bytes _s);
-/// Remove all '0' on the head of _s. Returns 0 if _s contains only '0'.
-std::string unpadLeft(std::string _s);
```

Comment:
this alteration of a base api function doesn't break stuff in the rest of the project?